owncloudApp/src/main/java/com/owncloud/android/ui/activity/WhatsNewActivity.java

```diff
@@ -58,13 +60,16 @@ public class WhatsNewActivity extends FragmentActivity implements ViewPager.OnPa
     private ProgressIndicator mProgress;
     private ViewPager mPager;
 
+    private WhatsNewActivityBinding binding_activity;
+    private static WhatsNewElementBinding binding_element;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.whats_new_activity);
 
-        mProgress = findViewById(R.id.progressIndicator);
-        mPager = findViewById(R.id.contentPanel);
+        mProgress = binding_activity.progressIndicator;
+        mPager = binding_activity.contentPanel;
```

Comment:
Neither `binding_activity` nor `binding_element` have been initialized, they were only declared 😉. Before the `setContentView` line, you should initialize them:
```
binding_activity = WhatsNewActivityBinding.inflate(getLayoutInflater());
```
Same with the other variable, I won't write the answer for you to practise 😁